### PR TITLE
Adding Serbian translation

### PR DESCRIPTION
**Pull request by Milos5611**
Hi there,

I'd kindly ask for responsibility of translating React.js to Serbian. I @Milos5611 and @vaske, we don't have previous experience in translating on this level but we are native Serbian speaker and use English professionally as our day-to-day language :-)

If there are any other fellow Serbian is willing to join, will be more than welcome! :)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ repo/*
 npm-debug.log
 yarn-error.log
 !.gitkeep
+reactjs.org-translation.iml
```
Comment (tesseralis) [inline .gitignore]: What's this line of code?

**Comment by tesseralis** [comment]
Sure! @vaske, could you confirm that you'd like to be a maintainer? Once you've done so, I'll create the repo.

**Comment by vaske** [comment]
@tesseralis yeah confirming ;)

File: .gitignore (hunk 1 above)
Comment (Milos5611) [reply .gitignore]: Removed, IntelliJ added it's config to project root instead `.idea` default config dir.

**Comment by tesseralis** [comment]
Here's your repo! https://github.com/reactjs/sr.reactjs.org

**Comment by DedaDev** [comment]
If you need any help, feel free to add me ;)
My experience: Was a lead translator for Serbian for Discord App.

**Pull request merged**